auth_backends/pipeline.py

```diff
@@ -3,8 +3,10 @@
 For more info visit http://python-social-auth.readthedocs.org/en/latest/pipeline.html.
 """
 from django.contrib.auth import get_user_model
+import logging
+from edx_django_utils.monitoring import set_custom_attribute
```

Comment:
I'm guessing you might have sorting issue here. Not sure if isort is running. I'm not sure if what I added below is correct, but it may be closer.
```suggestion
import logging
from django.contrib.auth import get_user_model

from edx_django_utils.monitoring import set_custom_attribute
```

Reply:
This can be marked resolved.